kafka-scheduler/src/main/java/com/mesosphere/dcos/kafka/offer/OfferUtils.java

```diff
@@ -21,4 +23,85 @@ public static String brokerIdToTaskName(Integer brokerId) {
   public static int nameToId(String brokerName) {
     return Integer.parseInt(brokerName.substring(brokerName.indexOf('-') + 1));
   }
+  
+  public static String getKafkaJmxOpts(JmxConfig heapConfig) {
```

Comment:
`OfferUtils` doesn't seem like the right location for this. Maybe make a class over in the `com.mesosphere.dcos.kafka.config` package.